### PR TITLE
setup teardown api test

### PR DESCRIPTION
**Pull request by sebastus**
### What does this PR do?
Add database tear down step to API test. 

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?

### CODE REVIEW

File: e2e/__tests__/confirm_API_data_browse.spec.ts [hunk 1]
```diff
@@ -16,5 +18,14 @@ test('API Datasource Browsing', async ({ page }) => {
   await page.goto('/');
   const elements = await page.locator('[id="TestAPIDataSource"]'); // After first run there will be more than one element by this name
   await elements.nth(0).click(); // Only look for the first element
-  await expect(page.getByText('AUTHOR')).toBeVisible();
+  await expect(page.getByText('Author')).toBeVisible();
```
Comment (hannahlspencer) [inline e2e/__tests__/confirm_API_data_browse.spec.ts:21]: Just out of interest, what's this line for?

File: e2e/__tests__/confirm_API_data_browse.spec.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import { test, expect } from '@playwright/test';
+import * as mongoDB from 'mongodb';
```
Comment (hannahlspencer) [inline e2e/__tests__/confirm_API_data_browse.spec.ts:2]: Really picky personal preference but I tend to prefer to name what's being imported, is there a reason you've gone with the wildcard?

**Review by hannahlspencer — APPROVED** [review]
Couple of comments/questions but otherwise looks good!

File: e2e/__tests__/confirm_API_data_browse.spec.ts (hunk 2 above)
Comment (777arc) [reply e2e/__tests__/confirm_API_data_browse.spec.ts:2]: agreed

File: e2e/__tests__/confirm_API_data_browse.spec.ts (hunk 1 above)
Comment (sebastus) [reply e2e/__tests__/confirm_API_data_browse.spec.ts:21]: If the control is visible, the page must be loaded.

File: e2e/__tests__/confirm_API_data_browse.spec.ts (hunk 2 above)
Comment (sebastus) [reply e2e/__tests__/confirm_API_data_browse.spec.ts]: fixed

**Pull request merged**